.github/workflows/tests-schedule.yml

```diff
@@ -0,0 +1,45 @@
+name: tests
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/tests-schedule.yml"
+      - "test/test_datasets_download.py"
```

Comment:
If a PR changes on of these files, we run these tests off-schedule to see if the tests are going through at all. Since this should not be happening often, I think this OK.